scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/JobsView.java

```diff
@@ -131,7 +131,7 @@ protected Widget getBuiltFilterPane() {
 
     @Override
     protected Label getLabel() {
-        Label label = new Label("Use filters to restrict the number of jobs currently displayed.<br>");
+        Label label = new Label("Use filters to restrict the number of jobs currently displayed.<br>Be aware that <b> using multiple filters on the same criteria</b> (id, priority,..) lead in most cases to <b>only consider your last filter</b>.");
```

Comment:
Be aware that <b> using multiple filters on the same criteria</b> (id, priority,..) leads, in most cases, to <b>only consider your last filter</b>. Orth.

Reply:
Ok.